.spelling

```diff
@@ -1,4 +1,5 @@
 256-bit
```

Comment:
noticed I screwed up a few prs back so I fixed the order

Reply:
I did sort it back here https://github.com/cirruslabs/cirrus-ci-docs/commit/d56a6de9298470420c5215b0ef27e0b27b4352fd#diff-b9a2754b5f195f0113dc72bd305fcdb3

Reply:
Strange - lots of things are out of order if you look at this PRs diff

Reply:
Are you sure these changes are needed? I don't see much new words except maybe *Maven* and *Junit*. Why there are so many changes when master is green at the moment.